.travis.yml

```diff
@@ -24,10 +24,10 @@ cache:
 
 before_install: . ./ci/before_install.sh
 
+before_script: cd libbindgen
+
 script:
-  - cargo build --verbose --features "$BINDGEN_FEATURES"
   - cargo test --features "$BINDGEN_FEATURES"
-  - cargo build --release --verbose --features "$BINDGEN_FEATURES"
   - cargo test --release --features "$BINDGEN_FEATURES"
```

Comment:
I do like this idea, though there's some weirdness going on. What is a "package" to `cargo test`?

* `clap` is a direct dependency of `bindgen` (which I had already built, as it happens)
* `clap` is a dev-dependency of the `libbindgen` crate

```
jdub@sel4:/tmp/rust-bindgen$ cargo test --features llvm_stable -p libbindgen
   Compiling libbindgen v0.17.0 (file:///tmp/rust-bindgen/libbindgen)
error[E0463]: can't find crate for `clap`
 --> libbindgen/tests/tests.rs:1:1
  |
1 | extern crate clap;
  | ^^^^^^^^^^^^^^^^^^ can't find crate

error: aborting due to previous error

error: Could not compile `libbindgen`.

To learn more, run the command again with --verbose.
```